nautobot/core/templates/inc/object_details_advanced_panel.html

```diff
@@ -42,6 +44,15 @@
                         </td>
                     </tr>
                 {% endif %}
+            </tbody>
+        </table>
+    </div>
+    <div class="panel panel-default">
```

Comment:
From Glenn: Looks weird to me to have "Created" in one panel and "Created By" in a different panel. Wonder if we can/should move the "Created" and "Last Updated" entries into the "Data Provenance" panel as well - any thoughts @lampwins?